test_runner/src/test/kotlin/ftl/util/ObfuscationGsonTest.kt

```diff
@@ -0,0 +1,21 @@
+package ftl.util
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+
+class ObfuscationGsonTest {
+
+    @Test
+    fun `Should return gson with correct type adapters registered`() {
```

Comment:
Everywhere we starting test name from lower letter. Maybe we should keep always same naming convention?

Reply:
updated